aries-site/src/pages/components/pageheader.mdx

```diff
@@ -0,0 +1,220 @@
+import {
+  ChildPageHeaderExample,
+  PageHeaderActions,
+  PageHeaderAnatomy,
+  PageHeaderContentRegions,
+  PageHeaderIntroExample,
+  PageHeaderResponsiveActions,
+  PageHeaderSubtitle,
+  TopLevelPageHeaderExample,
+} from '../../examples';
+import { Box, Grid } from 'grommet';
+
+The PageHeader's title communicates the context of the page. The subtitle elaborates secondary information about the page. The user is also able to access navigation to the parent page and any relevant page-level actions.
+
+<Example
+  docs="https://v2.grommet.io/pageheader?theme=hpe"
+  height={{ min: 'small' }}
+  showResponsiveControls={[]}
+  template
+>
+  <PageHeaderIntroExample />
+</Example>
+
+## Anatomy
+
+<Example
+  caption="Diagram illustrating the four main content regions of a PageHeader."
+  plain
+>
+  <PageHeaderAnatomy />
+</Example>
+
+1. **Page Identifier:** This region is intended to provide the user with context on what the page is about.
+
+   - **a.** [Title](#title) (required): A short and descriptive title that uniquely identifies the page.
+   - **b.** [Subtitle](#subtitle) (optional): Concise contextual information that helps elaborate the purpose of the page.
+
+2. [Contextual Navigation](#reverse-anchor): A Reverse Anchor is required on all child pages as a means of navigating up the information hierarchy.
+3. [Page-Level Actions](#page-level-actions) (optional): This region is reserved for actions that affect the entire page or kick-off a task flow.
+
+## Responsive Behavior and Content Prioritization
+
+The main purpose of a page header is to identify a page and its context within an application. Therefore, the title and subtitle regions should receive screen priority when they come into contact with page-level actions.
+
+The region containing the title and subtitle should:
+
+- Flex to fill the available horizontal space on the page.
+- Have a minimum width of "medium" on `medium-xlarge` breakpoints and "small" on `xsmall-small` breakpoints to avoid excessive wrapping.
+
+Page-level actions should:
+
+- Have a width of "auto" by default.
+- Progressively collapse into an overflow menu as screen width becomes limited. This should follow guidance on [responsive behavior of actions](#responsive-behavior-of-actions).
+
+Resize your screen to see how the content regions adapt at various screen widths.
+
+<Example showResponsiveControls={[]} screenContainer plain width="100%">
+  <PageHeaderContentRegions background="background-front" />
+</Example>
+
+## Title
+
+A title is a required element of the page and should:
+
+- Always be an h1 with size="small".
+- Uniquely and concisely identify the page. No two pages under the same parent should have the same name.
+- Follow title case capitalization (i.e., List of Clusters, My Account).
+- Wrap if too long for the available space. See [Responsive Behavior and Content Prioritization](#responsive-behavior-and-content-prioritization) for more guidance.
+- Be left-aligned with page content.
```

Comment:
since the grommet component takes care of the left-aligned do we need to add this ?

Reply:
I think we have it here for added clarity/criteria in case teams are designing things that don't follow our pattern, we can point to this as a "here is what the styling should be"

Reply:
okay thats fair